### ISSUE TITLE
iOS 4.2 simulator does not honor SBFake* preferences

### ISSUE THREAD

**Issue by mspasov** [lorenbrichter/SimFinger#8]
iOS 4.2 simulator does not honor SBFakeCarrier, SBFakeTime and SBFakeTime preferences (tried placing them in several locations)

**Comment by buholzer** [comment]
Any updates, have the same issue...

**Comment by thedigitalyogi** [comment]
Yeah - me too. And I can't load the fake apps either. Is anyone able to?

**Comment by HBehrens** [comment]
FakeCarrier and FakeTime is only recognized upto iOS Version 4.0.2.
Regarding the fake apps, you have to place it directly to the applications folder. Both should be fixed with 8f9c99b0ca4de5b4ba71be13fd896c68d0fe28b8

**Comment by brunogama** [comment]
someone can explain me how to put the fake apps manually in the Applications folder? I need to record a video today and i'm kinda lost.
